XtendInstallDebugInfoMojoIT.simpleSMAP is flaky

[ERROR] Failed to execute goal org.eclipse.xtend:xtend-maven-plugin:IT-SNAPSHOT:compile (default) on project simple: Execution default of goal org.eclipse.xtend:xtend-maven-plugin:IT-SNAPSHOT:compile failed: Plugin org.eclipse.xtend:xtend-maven-plugin:IT-SNAPSHOT or one of its dependencies could not be resolved: Failed to collect dependencies at org.eclipse.xtend:xtend-maven-plugin:jar:IT-SNAPSHOT -> org.eclipse.emf:org.eclipse.emf.codegen:jar:2.19.0 -> org.eclipse.jdt:org.eclipse.jdt.launching:jar:[3.5.0,4.0.0): No versions available for org.eclipse.jdt:org.eclipse.jdt.launching:jar:[3.5.0,4.0.0) within specified range -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.eclipse.xtend:xtend-maven-plugin:IT-SNAPSHOT:compile (default) on project simple: Execution default of goal org.eclipse.xtend:xtend-maven-plugin:IT-SNAPSHOT:compile failed: Plugin org.eclipse.xtend:xtend-maven-plugin:IT-SNAPSHOT or one of its dependencies could not be resolved: Failed to collect dependencies at org.eclipse.xtend:xtend-maven-plugin:jar:IT-SNAPSHOT ->

--- a/org.eclipse.xtend.maven.plugin/pom.xml
+++ b/org.eclipse.xtend.maven.plugin/pom.xml
@@ -132,6 +132,12 @@
 			<groupId>org.eclipse.emf</groupId>
 			<artifactId>org.eclipse.emf.codegen</artifactId>
 			<scope>runtime</scope>
+			<exclusions>
+				<exclusion>
+					<groupId>org.eclipse.jdt</groupId>
+					<artifactId>org.eclipse.jdt.launching</artifactId>
+				</exclusion>
+			</exclusions>
 		</dependency>
 	</dependencies>
 	<build>

